src/hotspot/share/compiler/compilerThread.cpp

```diff
@@ -55,8 +55,12 @@ CompilerThread::~CompilerThread() {
 }
 
 void CompilerThread::set_compiler(AbstractCompiler* c) {
-  // Only jvmci compiler threads can call Java
-  _can_call_java = c != nullptr && c->is_jvmci();
+  /*
```

Comment:
The comment could be a little shorter:
```
  /*
   * Compiler threads need to make Java upcalls to the jargraal compiler.
   * Java upcalls are also needed by the InterpreterRuntime when using jargraal.
   */
```

Reply:
Resolved in https://github.com/openjdk/jdk/pull/21285/commits/7e0f1a4227f388dc8e22e6200dc026f056d26eed.